app/views/refinery/authentication/devise/admin/users/_user.html.erb

```diff
@@ -1,23 +1,30 @@
-<li id="sortable_<%= user.id %>" class='clearfix record <%= cycle("on", "on-hover") %>'>
-  <span class='title'>
-    <strong><%= user.username %></strong>
-    <span class="preview">
-      <%= t('.preview', :who => user.email, :created_at => l(user.created_at, :format => :short)) %>
+<li class='page-listing' id="<%= dom_id(user) -%>">
+  <div class="page-row">
+    <i class="icon icon-user-1"></i>&nbsp;
+
+    <% tag_name = current_refinery_user.can_edit?(user) ? :a : :div %>
+    <%= content_tag(tag_name, class: 'page-title', href: refinery.edit_authentication_devise_admin_user_path(user)) do %>
+      <span class='title'>
+        <%= user.username %>
+        <span class='label'>
+          (<%= user.email %>)
+        </span>
+      </span>
+    <% end %>
+
+    <span class='actions'>
+      <% if current_refinery_user.can_delete?(user) %>
+        <span class="delete pull-right">
+          <%= link_to refinery.authentication_devise_admin_user_path(user),
+                      :class => "cancel confirm-delete circle-icon delete",
+                      :method => :delete,
+                      :data => {
+                        :confirm => t('message', :scope => 'refinery.authentication.devise.admin.delete', :title => user.username)
+                      } do %>
+            <i class="icon icon-trash bump-up-2"></i>
```

Comment:
It use the old way to add icon and the trash is not CSS centered (see my screenshot in comments)